Image was not panning while when triggering min zoom

palmithor opened the pull request:
This pull requests fixes the issue: https://github.com/jayrambhia/CropperNoCropper/issues/10

palmithor committed:
Image was not panning while when triggering min zoom - Fix for issue: https://github.com/jayrambhia/CropperNoCropper/issues/10

palmithor committed:
Cropping to center  should still be restricted the minimum zoom set. Change adds animation when cropToCenter exceeds the allowed aspect.

palmithor committed:
Accidentally added the adjustment with scale to cropToCenter() instead of fitToCenter() to begin with, now it is reversed

jayrambhia commented:
Thanks for the contribution. But there are so many changes (due to auto formatting and other stuff), I can't find what's the change that made it work. Can you explain what changes did you make to fix the issue?

palmithor commented:
in onUP in CropperImageView I changed one if condition.
From: 
```java
if (scaleX <= mMinZoom) 
// to:
if (scaleX < mMinZoom)
```

Otherwise the image was not panning correctly with min zoom when zooming far out.

I also added this to the end of fitToCenter() in the same class:
```java
// If over scrolled, return back to the place.
        float tx = getMatrixValue(matrix, Matrix.MTRANS_X);
        float ty = getMatrixValue(matrix, Matrix.MTRANS_Y);
        float scaleX = getMatrixValue(matrix, Matrix.MSCALE_X);
        if (scaleX < mMinZoom) {
            float xx = getWidth() / 2 - mMinZoom * drawable.getIntrinsicWidth() / 2;
            float yy = getHeight() / 2 - mMinZoom * drawable.getIntrinsicHeight() / 2;
            animateAdjustmentWithScale(tx, xx, ty, yy, scaleX, mMinZoom);
        }
```

The reason I added this is that I think that if the user has preset a min zoom, the view should not allow setting the zoom level lover than that which can happen on fitToCenter. So I animated it to minZoom.

jayrambhia commented:
Added code.
